packages/cursorless-engine/src/languages/LanguageDefinition.ts

```diff
@@ -136,10 +134,10 @@ async function readQueryFileAndImports(
       }
 
       if (doValidation) {
-        validateQueryCaptures(fileName, rawQuery);
+        validateQueryCaptures(filename, rawQuery);
       }
 
-      rawQueryStrings[queryPath] = rawQuery;
+      rawQueryStrings[filename] = rawQuery;
```

Comment:
I'm tempted to simplify this regex. Remove support for `/` and don't support `include`